src/routes/Game.svelte

```diff
@@ -48,7 +48,6 @@
   export let game_name;
   $: activeGame.set(game_name);
 
-  $: gameInBeta = $activeGame === SupportedGame.Jak3;

```

Comment:
I don't think this cross-game "is in beta?" variable is really that useful, instead I'm just checking below which game we're on directly and displaying the matching beta message from that